src/Estimators/src/LeggedOdometry.cpp

```diff
@@ -917,7 +917,7 @@ bool LeggedOdometry::Impl::changeFixedFrame(const iDynTree::FrameIndex& newIdx,
 
 
 bool LeggedOdometry::changeFixedFrame(const std::ptrdiff_t& newIdx,
-                                      const Eigen::Quaterniond& frameOrientationInWorld,
```

Comment:
As I mentioned above you may avoid converting to change from quaternion to vector4 since you can apply the modification directly to python